internal/fs/inode/dir.go

```diff
@@ -580,7 +582,24 @@ func (d *dirInode) ReadDescendants(ctx context.Context, limit int) (map[Name]*Co
 			return descendants, nil
 		}
 	}
+}
+
+func logUnsupportedListings(removedListings *gcs.Listing) {
```

Comment:
> shall we log unsupported objects in the util method itself

I thought of doing that too at first, but that'd require me to create a dependency of util package on logger/config, which is a cyclic dependency. To avoid that, I resorted to this alternative.